test/python/tf_unittest_runner.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import sys
+import argparse
+import os
+
+#Import ngraph in builds without ngraph embedded
+try:
+    import ngraph
+except:
+    pass
+
+loader = unittest.TestLoader()
+parser = argparse.ArgumentParser()
+optional = parser._action_groups.pop() 
+required = parser.add_argument_group('required arguments')
+required.add_argument('--tensorflow_path', help="Specify the path where Tensorflow is installed. Eg:/localdisk/skantama/tf-ngraph/tensorflow \n", required=True)
+optional.add_argument('--list_tests', help="Prints the list of test cases in this package. Eg:math_ops_test \n")
+optional.add_argument('--run_test', help="Runs the testcase and returns the output. Eg:math_ops_test.math_ops_test.DivNoNanTest.testBasic")
+parser._action_groups.append(optional)
+arguments = parser.parse_args()
+
+def main():
+    all_dirs(arguments.tensorflow_path)
+
+    if(arguments.list_tests):
+        list_tests()
+
+    if(arguments.run_test):
+        run_test()
+
+from fnmatch import fnmatch
+def all_dirs(dirname):
+    dirname = arguments.tensorflow_path
+    pattern = "*_test.py"
+    for path, subdirs, files in os.walk(dirname):
+        for name in files:
+            if fnmatch(name, pattern):
+                sys.path.append(path)
+
+def list_tests():
+    test_module = arguments.list_tests
+    module = __import__(test_module)
+    test_modules = loader.loadTestsFromModule(module)
+    alltests = []
+    for test_class in test_modules:
+        alltests.append( ([i.id() for i in test_class._tests]))
+    print ('\n'.join((sorted(sum(alltests, [])))))
+
+
+def run_test():
+    test_name = arguments.run_test
+    tests = loader.loadTestsFromName(test_name)
+    test_result = unittest.TextTestRunner(verbosity=2).run(tests)
```

Comment:
Lets also have `def run_test(... verbosity=2)` as an argument.
In which case:
`unittest.TextTestRunner(verbosity=verbosity).run(tests)`